.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install sphinx
           pip install tox
```

Comment:
Hello @tsolakoua, can ```pip install -r requirements.txt``` be used here? As different Sphinx versions are being used for the workflow otherwise.

Reply:
Thanks a lot @siddydutta! I added the same versions in both requirements.txt and GitHub Actions workflow and it now doesn't fail.